workloads/network-perf/run_serviceip_network_test_fromgit.sh

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env bash
+set -x
+
+_es=search-cloud-perf-lqrf3jjtaqo7727m7ynd2xyt4y.us-west-2.es.amazonaws.com
+_es_port=80
+
+if [[ ${ES_SERVER} ]]; then
+  _es=${ES_SERVER}
+fi
+
+if [[ ${ES_PORT} ]]; then
+  _es_port=${ES_PORT}
+fi
+
+kubeconfig=$2
+if [ "$cloud_name" == "" ]; then
+  kubeconfig="$HOME/kubeconfig"
+fi
+
+cloud_name=$1
+if [ "$cloud_name" == "" ]; then
+  cloud_name="test_cloud"
+fi
+
+echo "Starting test for cloud: $cloud_name"
+
+oc create ns my-ripsaw
+
+git clone http://github.com/cloud-bulldozer/ripsaw /tmp/ripsaw
+oc apply -f /tmp/ripsaw/deploy
+oc apply -f /tmp/ripsaw/resources/crds/ripsaw_v1alpha1_ripsaw_crd.yaml
+oc apply -f /tmp/ripsaw/resources/operator.yaml
+
+server=""
+client=""
+pin=false
+if [[ $(oc get nodes | grep worker | wc -l) -gt 1 ]]; then
+  server=$(oc describe nodes/$(oc get nodes | grep worker | tail -1 | awk '{print $1}') | grep hostname | awk -F= '{print $2}')
+  client=$(oc describe nodes/$(oc get nodes | grep worker | head -1 | awk '{print $1}') | grep hostname | awk -F= '{print $2}')
+  pin=true
+fi
+
+# Create Service Account with View privileges for backpack
+oc delete ClusterRoleBinding/backpack-view
```

Comment:
We can use ripsaw's YAML files to create ClusterRoleBinding and serviceAccount, as these ones don't inherit any parameters

Reply:
Hm, not sure about that? @dry923 thoughts?

Reply:
Ah ha! I see you point now. Fixing!